cmd/lotus-provider/stop.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	_ "net/http/pprof"
+
+	"github.com/urfave/cli/v2"
+
+	lcli "github.com/filecoin-project/lotus/cli"
+)
+
+var stopCmd = &cli.Command{
```

Comment:
Does this command actually call lotus-provider? I believe it doesn't even serve RPC right now.